spec/rubocop/cop/rails/presence_spec.rb

```diff
@@ -128,6 +128,32 @@
     CORRECTION
   end
 
+  context 'when a right-hand side of the relational operator' do
+    %w[< > <= >= == !=].each do |operator|
+      it "registers an offense and corrects when `#{operator}`" do
+        source = <<~SOURCE.chomp
+          a #{operator} if b.present?
+            b
+          else
+            c
+          end
+        SOURCE
+        current = <<~CURRENT.chomp
+          if b.present?
+            b
+          else
+            c
+          end
+        CURRENT
+        offenses = inspect_source(source)
+        expect(offenses.count).to eq 1
+        expect(offenses).to all(have_attributes(first_line: 1, last_line: 5))
+        expect(offenses).to all(have_attributes(message: "Use `(b.presence || c)` instead of `#{current}`."))
+        expect(autocorrect_source(source)).to eq("a #{operator} (b.presence || c)")
+      end
```

Comment:
This test code looks complicated. Can you update the test to be simpler?

Reply:
I opened the following Pull Request because We want to make some improvements first to simplify the test.
- https://github.com/rubocop/rubocop-rails/pull/817

Reply:
Thank you for the review and merge. We have changed the test to a simple.